### PR TITLE
Create githubdisruptivestudent.md

### PR DESCRIPTION
**Pull request by CIS-BDooley**
This pull request adds a bio file for @CIS-Bdooley as requested in #3 

@githubteacher 
Hey @pprabhala, I would :heart_decoration: if you would review my proposed changes.

### CODE REVIEW

**Comment by CIS-BDooley** [comment]
This is yet another comment. :8ball:

File: docs/githubdisruptivestudent.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+**GITHUB Disruptive Student**
+- Name: Bill
+- Occupation: Manager
+- Preferred Libation: Guinness
```
Comment (kpodonog) [inline docs/githubdisruptivestudent.md]: Not happy with filppancy :angry:

Comment (CIS-BDooley) [inline docs/githubdisruptivestudent.md]: What? Not Loose Cannon?

Comment (iamKurt) [inline docs/githubdisruptivestudent.md]: Guinness, like the :beer:?

**Comment by CIS-BDooley** [comment]
This fixes #3

**Comment by CIS-BDooley** [comment]
This fixes #3

**Pull request merged**